protocol-units/types/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub use aptos_api::runtime::Apis;
+pub use aptos_crypto::hash::HashValue;
+pub use aptos_types::{
+	block_executor::partitioner::ExecutableBlock,
+	block_executor::partitioner::ExecutableTransactions,
+	transaction::signature_verified_transaction::SignatureVerifiedTransaction,
+	transaction::{SignedTransaction, Transaction},
+};
+
+pub mod executor;
+pub mod finality_mode;
+
+pub use executor::ExecutorOps;
+pub use finality_mode::FinalityMode;
```

Comment:
We've talked with @l-monninger about `FinalityMode`; it's a leftover from an older design and should be removed. Maybe not in this PR, though.

Reply:
I was not aware. Will leave it in for now.